interfaces/lego-boost/tools/motion/js/robotdummy.js

```diff
@@ -7,12 +7,35 @@ export class RobotDummy extends THREE.Group {
 
         super();
 
-        let geometryMarker = new THREE.BoxGeometry( 170, 70, 290 );
-        let materialMarker = new THREE.MeshBasicMaterial( {color: 0x836490, transparent: true, opacity: 0} );
+        /*let geometryMarker = new THREE.BoxGeometry( 170, 70, 290 );
+        let materialMarker = new THREE.MeshBasicMaterial( {color: 0x836490, transparent: true, opacity: 1} );
 
         let robotMarker = new THREE.Mesh( geometryMarker, materialMarker );
 
-        this.add(robotMarker);
+        this.add(robotMarker);*/
+
+        // Robot dummy for Object Target
+        const geometrycube = new THREE.BoxGeometry( 10, 10, 10 );
+        const materialcube = new THREE.MeshBasicMaterial( {color: 0xffffff} );
+        let robotDummy = new THREE.Mesh( geometrycube, materialcube );
+        //robotDummy.matrixAutoUpdate = false;
+        robotDummy.position.set(0,0,0);
+        this.add( robotDummy );
+
+        const materialcube_x = new THREE.MeshBasicMaterial( {color: 0xff0000} );
```

Comment:
nit: the variables in this section would be more consistent as materialCube, materialCubeX, etc.